tests/pytest.ini

```diff
@@ -19,12 +19,5 @@ markers =
     logging: marks tests for pennylane logging
     external: marks tests that require external packages such as matplotlib and PyZX
     catalyst: marks tests for catalyst testing (select with '-m "catalyst"')
-filterwarnings =
-    ignore::DeprecationWarning:autograd.numpy.numpy_wrapper
-    ignore:Casting complex values to real::autograd.numpy.numpy_wrapper
-    ignore:Casting complex values to real discards the imaginary part:UserWarning:torch.autograd
-    ignore:Call to deprecated create function:DeprecationWarning
-    ignore:the imp module is deprecated:DeprecationWarning
-    error::pennylane.PennyLaneDeprecationWarning
```

Comment:
Probably good to keep the `PennyLaneDeprecationWarning` warning-to-error promotion. We should intercept that always, not just on weekly runs.

Reply:
Oops, missed the last one -- good catch. Yea, don't want that to be cut, just the ignores.